aries_cloudagent/anoncreds/default/did_web/registry.py

```diff
@@ -19,7 +19,9 @@ class DIDWebRegistry(BaseAnonCredsResolver, BaseAnonCredsRegistrar):
     """DIDWebRegistry"""
 
     def __init__(self):
-        self._supported_identifiers_regex = re.compile(r"^did:web:.*$")
+        self._supported_identifiers_regex = re.compile(
+            r"^(did:web:)([a-zA-Z0-9%._-]*:)*[a-zA-Z0-9%._-]+$"
```

Comment:
Would this pattern match a DID URL formed from a did:web? For example `did:web:example.com/anoncreds/v0/SCHEMA/asdf`

Reply:
good catch I will update that.